### PR TITLE
Spin assertSelectContainsExactOptions()

### PR DESCRIPTION
**Pull request by Chekote**
Note: This is the v2.0 fork of \#144

Problem:
I need to be able to test for options in a select that are added via an asynchronous process. The method currently fails immediately if the options are not present.

Fix:
I modified the method to use a waitFor()

### CODE REVIEW

**Comment by medology-bugman** [comment]
Paging @amberwjq to review the front-end work in this pull request!
Paging @juanmrad to review the back-end work in this pull request!
Paging @RedEnchilada to review the testing work in this pull request!

File: src/Medology/Behat/Mink/FlexibleContext.php [hunk 1]
```diff
@@ -481,45 +481,47 @@ public function assertSelectContainsExactOptions($select, TableNode $tableNode)
             throw new InvalidArgumentException('Arguments must be a single-column list of items');
         }
 
-        $expectedOptTexts = array_map([$this->storeContext, 'injectStoredValues'], $tableNode->getColumn(0));
-
+        $expectedOptTexts = array_map([$this, 'injectStoredValues'], $tableNode->getColumn(0));
```
Comment (RedEnchilada) [inline src/Medology/Behat/Mink/FlexibleContext.php]: `injectStoredValues` isn't present on this context. It looks like you ported the code over from the 1.0 branch without updating method references for the context sharing structure in 2.0?

**Comment by Chekote** [comment]
I noticed that none of the checks were marked as required, which is why Bugman assigned reviewers before this had passed CI. I've added the requirements now. Please hold off on review until the PR passes CI. Thanks.

**Comment by Chekote** [comment]
I'm closing this PR because it does not make sense for FlexibleMink 2.0. The spinning functionality was removed from 2.0 into an external package Chekote/BehatRetryExtension.

**Pull request closed without merging**